config.go

```diff
@@ -894,6 +938,29 @@ func (cfg *Config) storageHasCertResources(issuer Issuer, domain string) bool {
 		cfg.Storage.Exists(metaKey)
 }
 
+// deleteSiteAssets deletes the folder in storage containing the
+// certificate, private key, and metadata file for domain from the
+// issuer with the given issuer key.
+func (cfg *Config) deleteSiteAssets(issuerKey, domain string) error {
```

Comment:
FWIW, I think this would be useful to expose, could allow for implementing a CLI command to wipe out assets for a site from CLI in Caddy (or via the admin API maybe).

Reply:
I like that idea. Will need to think about a good way to do this.

Reply:
Maybe 3rd param `fullWipe bool` or something?

Reply:
This'll have to come in another feature/PR.